CHANGELOG.asciidoc

```diff
@@ -3,30 +3,169 @@
 :issue: https://github.com/elastic/beats/issues/
 :pull: https://github.com/elastic/beats/pull/
 
-[[release-notes-8.0.0-rc2]]
-=== Beats version 8.0.0-rc2
+[[release-notes-8.0.0]]
+=== Beats version 8.0.0
+https://github.com/elastic/beats/compare/v7.17.0...v8.0.0[View commits]
 
-Changes will be described in the GA release.
+==== Breaking changes
+
+*Affecting all Beats*
+
+- Remove the deprecated `xpack.monitoring.*` settings. Going forward only `monitoring.*` settings may be used. {issue}9424[9424] {pull}18608[18608]
+- Remove deprecated/undocumented IncludeCreatorMetadata setting from kubernetes metadata config options {pull}28006[28006]
+- Remove deprecated fields from kubernetes module {pull}28046[28046]
+- Remove deprecated config option aws_partition. {pull}28120[28120]
+- Improve stats API {pull}27963[27963]
```

Comment:
This is vague and does not look like a breaking change. How about this and moving it to enhancements.

```suggestion
- Improve stats API by adding host metadata. {pull}27963[27963]
```